packages/ssh/test.R

```diff
@@ -0,0 +1,2 @@
+options(download.file.method="curl")
+install.packages("ssh", repos="https://cran.rstudio.com")
```

Comment:
Would be good to do more than just load the package, e.g.,

```
libssh_version()
```

Reply:
Has been added to `test.R`